### DISCUSSION TITLE
Github Copilot Nightly can't sign in

### DISCUSSION THREAD

**Discussion by almblr** [community/community#59321] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi, I restored my Windows 11 yesterday and I reinstalled all my programs including VSCode Insiders. After that, I reinstalled all my extensions and all of them works except Github Copilot Nightly, no matter which version I use.

The first left bottom pop-up asked me to Sign-in to use Github Copilot. I click on "Sign in to Github" then I have a message: "Allow an extension to open this URI? Github Authentication (vscode.github-authentication) wants to open a URI: vscode-insiders://vscode.githu.. ...253D1"

I click on "open" button then nothing happens. The copilot icon on the bottom is still an infinite spinner. So I can't use copilot during my work.

Does anyone know what I can do to fix that? Thanks

**Comment by Aceandra** [comment]
I have an issue presenting the exact same way.

I'm trying to sign in on a separate machine and am currently unable to do so, regardless of whether I use Nightly or Stable. Is there an ongoing outage of some kind?

**Reply by almblr** [reply]
I am reassured to see that I am not the only one, hope this will be fixed soon :(

**Reply by mrbjjackson** [reply]
I've just downloaded the VSCode - Insiders version to use Github Copilot Nightly and I can't get it to authenticate... Hopefully this is just a temporary thing...

**Reply by v1lton** [reply]
I am facing the same issue. I just can't sign in and there is no log. I am using MacOS.

**Comment by Aceandra** [comment]
After some further testing, I can confirm sign-in does not appear to be working at this time.

- Uninstalled VS Code and VS Code - Insiders
- Deleted %AppData%/Code and %AppData%/Code - Insiders
- Deleted %UserProfile%/.vscode and %UserProfile%/.vscode-insiders
- Reinstalled Code - Insiders
- Installed *only* the Copilot NIghtly and Copilot Chat extensions
- Tried to sign in to GitHub - failed as per the symptoms described earlier in this thread

EDIT: I also tried revoking VS code from my GitHub account several times. This, too, did not help.

**Comment by codewithpassion** [comment]
I've downgraded to 1.80.0.20230623 using chocolatey (https://community.chocolatey.org/packages/vscode-insiders/1.80.0.20230623) and that allowed me to sing in. So it looks like the latest version (1.80.0.20230626) has an issue. 

Other extensions, like Github Actions also had the same issue for me.

**Reply by Aceandra** [reply]
Confirming that this works, and the problem appears to be with the latest build of Insiders.

**Reply by almblr** [reply]
Works for me too. Thanks a lot

**Comment by glafarge** [comment]
Same issue.
Resolved by temporary downgrade using the extension menu

![image](https://github.com/community/community/assets/670645/68850c19-720c-4e16-b4ca-d5ae3295e9ac)

**Comment by Aceandra** [comment]
The latest version of Code - Insiders should now work with the latest version of Copilot Nightly.
